src/AssetsTransferApi.ts

```diff
@@ -186,7 +227,12 @@ export class AssetsTransferApi {
 
 		let txMethod: Methods;
 		let transaction: SubmittableExtrinsic<'promise', ISubmittableResult>;
-		if (assetType === AssetType.Foreign) {
+		if (
+			assetType === AssetType.Foreign ||
+			(assetType === AssetType.Native &&
```

Comment:
Can we abstract this second conditional after the `||` and assign it to a variable. I think it would make it a lot more readable.

Reply:
good catch! Ill do that and assign it to a variable :)